Auth0/Authentication.swift

```diff
@@ -712,7 +847,7 @@ public protocol Authentication: Trackable, Loggable {
      - [Authentication API Endpoint](https://auth0.com/docs/api/authentication#refresh-token)
      - [Refresh Tokens](https://auth0.com/docs/secure/tokens/refresh-tokens)
      */
-    func ssoExchange(withRefreshToken: String) -> Request<SSOCredentials, AuthenticationError>
```

Comment:
This is to address a DoCC warning:

<img width="1069" alt="Screenshot 2025-04-28 at 12 56 12" src="https://github.com/user-attachments/assets/e4e75ed6-9e5e-4979-8233-e02870b57d54" />

Since this is the *internal* parameter (the *external* one is `withRefreshToken`), it has no bearing on the public API.

Reply:
`refreshToken` is already the internal parameter in the actual implementing type.